### ISSUE TITLE
[feature request] add a "timeout waiting for js" option

### ISSUE THREAD

**Issue by jaredly** [juliangruber/tape-run#6]
If browserify has an error and the js doesn't get piped in, everything just hangs...

Example:

```bash
$ browserify missing.js|tape-run
Error: Cannot find module './missing.js' from '...'
    at /usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:55:21
    at load (/usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:82:15)
```

It would be nice to have a configurable timeout on that.

**Comment by juliangruber** [comment]
I found a better solution: When no javascript is written to it, an error is thrown. In the example above, there's only the warning printed to stderr, but stdout is empty.

Starting at version `1.1.0`, this will happen:

```bash
 ∴  tape-run (master) : browserify missing.js | tape-run
Error: Cannot find module '/Users/julian/dev/juliangruber/tape-run/missing.js' from '/Users/julian/dev/juliangruber/tape-run'
    at /usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:55:21
    at load (/usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:108:15)

stream.js:94
      throw er; // Unhandled stream error in pipe.
            ^
Error: javascript required
    at Stream.<anonymous> (/Users/julian/dev/juliangruber/tape-run/node_modules/browser-run/index.js:25:34)
    at _end (/Users/julian/dev/juliangruber/tape-run/node_modules/browser-run/node_modules/through/index.js:61:9)
    at Stream.stream.end (/Users/julian/dev/juliangruber/tape-run/node_modules/browser-run/node_modules/through/index.js:70:5)
    at Stream.method [as end] (/Users/julian/dev/juliangruber/tape-run/node_modules/browser-run/node_modules/duplexer/index.js:51:39)
    at Stream.<anonymous> (/Users/julian/dev/juliangruber/tape-run/node_modules/throughout/index.js:7:25)
    at _end (/Users/julian/dev/juliangruber/tape-run/node_modules/through/index.js:65:9)
    at Stream.stream.end (/Users/julian/dev/juliangruber/tape-run/node_modules/through/index.js:74:5)
    at Stream.onend (stream.js:79:10)
    at Stream.emit (events.js:117:20)
    at drain (/Users/julian/dev/juliangruber/tape-run/node_modules/through/index.js:34:23)
 ∴  tape-run (master) : echo $?
8
```

**Comment by lzrski** [comment]
Hey, thanks for tape-run. I'm switching from Mocha to Tape and your tools are really cool to work with.

To the point. I have encountered this `javascript required` error few times already and did not understood what what it meant until I found this by pure chance while browsing the issues. I think the error message could be more verbose, something like:

```
Error: No javascript was provided for the browser.
```
